commands/debug/debug_info.go

```diff
@@ -169,12 +175,10 @@ func getDebugProperties(req *rpc.GetDebugConfigRequest, pme *packagemanager.Expl
 		}
 	}
 
+	toolchainPrefix := debugProperties.Get("toolchain.prefix")
 	// HOTFIX: for samd (and maybe some other platforms). We should keep this for a reasonable
```

Comment:
This is also for other cores like the esp32 one

Reply:
yes

Reply:
So I'd change the comment

Reply:
The comment says:

> for samd (and maybe some other platforms)

esp32 is some other platform :-)